airflow/jobs.py

```diff
@@ -1139,6 +1143,9 @@ def _execute_task_instances(self,
                     queue=queue)
 
                 open_slots -= 1
+                if pool:
+                    pool_percent_full = (1 - open_slots / pools[pool].slots) * 100
```

Comment:
since (1 - int/int) should produce an int, it might be worthwhile to cast `open_slots` as a float, to avoid precision errors.